tests/system/test_error_details.py

```diff
@@ -77,16 +82,23 @@ def create_precondition_failure_details():
     status = create_status(pf_details)
 
     with pytest.raises(exceptions.GoogleAPICallError) as e:
-        _ = echo.echo(showcase.EchoRequest(
-            error=status,
-        ))
-        assert e.details == [pf_details]
+        _ = echo.echo(
+            showcase.EchoRequest(
+                error=status,
+            )
+        )
+
+    # Note: gRPC errors expose e.value.details.
+    exc_details = e.details if "rest" in str(
+        echo.transport) else e.value.details
+    assert exc_details == [pf_details]
 
 
 def test_unknown_details(echo):
     status = create_status()
     with pytest.raises(exceptions.GoogleAPICallError) as e:
-        _ = echo.echo(showcase.EchoRequest(
-            error=status,
-        ))
-        assert e.details == status.details
+        _ = echo.echo(
+            showcase.EchoRequest(
+                error=status,
+            )
+        )
```

Comment:
Is it possible to keep the assertion? 

If it would be possible with non-trivial work, I suggest keeping it, but commenting it out and adding a TODO summarizing what it would take to re-enable it.

Reply:
Removed it because we were just comparing empty details but added it back now.